README.rst

```diff
@@ -82,6 +82,16 @@ Manipulate BLOBs like so::
     # deletion
     client.blob_delete(table_name, digest)
 
+Version matrix
+==============
++--------------+------------+
+| Crate Ruby   | CrateDB    |
++==============+============+
+| < 0.9        | < 0.57     |
++--------------+------------+
+| 0.9          | >= 0.57    |
++--------------+------------+
```

Comment:
just to be sure. that table name change (  `schema_name` -> `table_schema`) was introduced in version 0.57?

Reply:
@corck Yes, https://github.com/crate/crate/blob/0.57/CHANGES.txt#L224